agent_build/tools/common.py

```diff
@@ -45,6 +46,16 @@ def init_logging():
     )
 
 
+# Make shlex.join for Python 3.7
```

Comment:
Should we just run those tests under Python 3.8 (same version we use for Windows binaries and Docker images)?

Reply:
I actually started thinking that besides frozen binaries, we may want to provide a "last stand" tarball with source code, as we do it now, so it would be better to at least comply with currently supported version of the python. WDYT?

Reply:
If it's not tons of compatibility code I think it should be fine.